cypress/e2e/specialFeatures/serviceWorker/index.cy.js

```diff
@@ -0,0 +1,25 @@
+import runTestsForPage from '#nextjs/cypress/support/helpers/runTestsForPage';
+import {
+  serviceWorkerIsAvailable,
+  serviceWorkerIsRegistered,
+  serviceWorkerCaching,
+} from './assertions';
+
+const tests = [
+  serviceWorkerIsAvailable,
+  serviceWorkerIsRegistered,
+  serviceWorkerCaching,
+];
+
+const testSuites = [
+  {
+    path: '/pidgin/articles/czje40pxkypo?renderer_env=live',
```

Comment:
Do we need more examples of other page types / services?

Reply:
It should work the same for all of them. I think I might add a couple of serbian examples lat + cyr - just to make sure the routing works as expected? We could focus on articles first, but it should work for other page types too, which use the cached libraries. 

Also, because these tests live in the Express app, there are no tests for live (or any other NextJS rendered) pages - but this can perhaps come later?